app.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "MeiliSearch",
+  "description": "Instant search API",
+  "keywords": [
+    "productivity",
+    "HTML5",
+    "scalpel"
```

Comment:
Are you sure about these keywords?

Reply:
🙄 2 minutes

Reply:
You know that you can create draft PR? This way I will not review it until you make it a real PR?
Unfortunately you cannot switch to a Draft if you made a real one...

Reply:
I will do it for the next PRs